Formula/dirmngr.rb

```diff
@@ -13,6 +13,8 @@ class Dirmngr < Formula
     sha256 "47fe29be8ca19eeb4d4a3e3434cd35ef7b13e1c1a9e8696f5ebd4434dc8cc062" => :mavericks
   end
 
+  keg_only "GPG2.1.x ships an internal dirmngr which it must use."
```

Comment:
Copy-pasted out of the old gnupg21 formula:
```
  conflicts_with "gnupg2",
        :because => "GPG2.1.x is incompatible with the 2.0.x branch."
  conflicts_with "gpg-agent",
        :because => "GPG2.1.x ships an internal gpg-agent which it must use."
  conflicts_with "dirmngr",
        :because => "GPG2.1.x ships an internal dirmngr which it it must use."
  conflicts_with "fwknop",
        :because => "fwknop expects to use a `gpgme` with Homebrew/Homebrew's gnupg2."
  conflicts_with "gpgme",
        :because => "gpgme currently requires 1.x.x or 2.0.x."
```
CC @DomT4 for orthographical wisdom.